components/AppBar/README.md

```diff
@@ -76,9 +76,9 @@ The Material Design top app bar displays information and actions relating to the
   - [Enabling top layout guide adjustment](#enabling-top-layout-guide-adjustment)
   - [Enabling inferred top safe area insets](#enabling-inferred-top-safe-area-insets)
 - [Extensions](#extensions)
-  - [Color Theming](#color-theming)
-  - [Typography Theming](#typography-theming)
-  - [Theming Extensions (Beta)](#theming-extensions-(beta))
+  - [Color Theming (To Be Deprecated)](#color-theming-(to-be-deprecated))
+  - [Typography Theming (To Be Deprecated)](#typography-theming-(to-be-deprecated))
+  - [Theming Extensions](#theming-extensions)
```

Comment:
This should be moved above the other articles in the table of contents and renamed “Theming” to be made consistent with other components.

Reply:
Done